### PR TITLE
python3Packages.aiogithubapi: init at 21.8.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python client for the GitHub API

https://github.com/ludeeus/aiogithubapi

This is a Home Assistant dependency.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 136937` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.aiogithubapi</li>
    <li>python39Packages.aiogithubapi</li>
  </ul>
</details>

File: pkgs/development/python-modules/aiogithubapi/default.nix [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  src = fetchFromGitHub {
+    owner = "ludeeus";
+    repo = pname;
+    rev = version;
+    sha256 = "1l59bb956gymi2dkgg88mr0nc80wqvw2dv69qf4f1xr3jxy03qa2";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py --replace "main" ${version}
```
Comment (samuela) [inline pkgs/development/python-modules/aiogithubapi/default.nix:27]: Why is this necessary? I think this deserves an explanation in a comment.

**Review by samuela — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/aiogithubapi/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/aiogithubapi/default.nix]: We need the the current version in `setup.py`. I updated the `substituteInPlace` to make it clear that's about the version.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 136937` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python38Packages.aiogithubapi</li>
<li>python39Packages.aiogithubapi</li>
</ul>
</details>
